src/Components/ExperimentOne/ExperimentOneUserForm.js

```diff
@@ -10,6 +10,7 @@ function DisplayUserRecommendationBasedOnQuery(userActivityFromApi,userActivityT
 function ExperimentOneUserForm(props){
 
     const activityTypes=["social", "recreational", "busywork", "diy", "charity", "cooking", "relaxation", "music", "educational"];
+    const listOfParticipants=['1','2','3','4','5']
```

Comment:
Might make more sense to use numbers here rather than strings (and update the usages)
```suggestion
    const listOfParticipants=[1, 2, 3, 4, 5];
```

Reply:
Yeah, you are right. Will update this